flexmeasures/api/v3_0/users.py

```diff
@@ -33,6 +35,7 @@
 user_schema = UserSchema()
 users_schema = UserSchema(many=True)
 partial_user_schema = UserSchema(partial=True)
+account_schema = AccountSchema()
```

Comment:
I don't see this being used.

Reply:
Yeah, this has been removed, I'm still working on this currently, the last time I merged was after the conflict resolution, but I will be making a commit soon.

Reply:
It's still here though.

Reply:
I'm using it now